.github/workflows/build.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
 
       - uses: sigstore/cosign-installer@4079ad3567a89f68395480299c77e40170430341
         with:
-          cosign-release: 'v2.0.0-rc.0'
+          cosign-release: 'v2.0.0-rc.1' # optional
```

Comment:
why the comment?

Reply:
I am not sure the reasons, tbh. Let's remove it.